src/asynchro.rs

```diff
@@ -291,7 +291,11 @@ macro_rules! resampler_sincfixedin {
             ///
             /// The function returns an error if the length of the input data is not equal
             /// to the number of channels and chunk size defined when creating the instance.
-            fn process(&mut self, wave_in: &[Vec<$t>]) -> Res<Vec<Vec<$t>>> {
+            fn process_in_place(
```

Comment:
In-place usually means the output overwrites the input. I would call this out-of-place. But in a way that's what the plain "process" does too. How about "process_into_buffer" or some variation on that?

Reply:
Done